mne/viz/_brain/_brain.py

```diff
@@ -254,6 +254,11 @@ class Brain(object):
         Color of the foreground (will be used for colorbars and text).
         None (default) will use black or white depending on the value
         of ``background``.
+    theme : str | path-like
+        Can be "auto" (default), "light", or "dark" to use darkdetect
+        (required for auto mode) or qdarkstyle (required for dark mode)
+        to style the widgets. Automatic detection does not yet work on
+        Linux. Can also be a path-like to a custom style sheet.
```

Comment:
I would open an upstream issue with `darkdetect` for this if they don't have one. In the meantime I would use:
```
gsettings get org.gnome.desktop.interface gtk-theme
```
if it ends in `-dark` assume it's dark. It at least [works on my machine](https://blog.codinghorror.com/the-works-on-my-machine-certification-program/)™ :)

Reply:
@cbrnr opened an issue

Reply:
This one: https://github.com/albertosottile/darkdetect/issues/7